akvo/rsr/models/indicator.py

```diff
@@ -196,7 +196,7 @@ def children_aggregate_percentage(self):
         """
         if self.measure == '2' and self.is_parent_indicator() and \
                 self.result.project.aggregate_children and \
-                any([ind.result.project.aggregate_to_parent for ind in self.child_indicators.all()]):
+                any(self.child_indicators.values_list('result__project__aggregate_to_parent', flat=True)):
```

Comment:
Brilliant! Just one question, would it be even more performant if we prefetched child_indicators since that's the reverse of the Indicator.parent_indicator FK?